modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -629,17 +630,37 @@ private void _addOrUpdateObjectFieldSettings(
 					newObjectFieldSetting.getName());
 
 			if (oldObjectFieldSetting == null) {
-				_objectFieldSettingLocalService.addObjectFieldSetting(
-					objectField.getUserId(), objectField.getObjectFieldId(),
-					newObjectFieldSetting.getName(),
-					newObjectFieldSetting.getValue());
+				if (GetterUtil.getBoolean(
+						PropsUtil.get("feature.flag.LPS-156704")) &&
```

Comment:
@marcelabc can you suppress this FF from here? They should be only in the headless layer.